controllers/tests/array_action/test_settings.py

```diff
@@ -54,3 +54,5 @@
 DUMMY_IP_ADDRESS2 = "2.2.2.2"
 DUMMY_IP_ADDRESS3 = "3.3.3.3"
 DUMMY_IP_ADDRESS_6_1 = "1::1"
+
+DUMMY_IO_GROUP = '1:2:3'
```

Comment:
We have define for dummy io group in common settings:
DUMMY_IO_GROUP = "iogrp1"

Reply:
I changed his name and now it's not like the one in the common settings, is it ok?